appstore/validator.go

```diff
@@ -81,6 +82,7 @@ func New() Client {
 	client := Client{
 		URL:     SandboxURL,
 		TimeOut: time.Second * 5,
+		SandboxURL: SandboxURL,
```

Comment:
Why does the client needs to keep the reference to "SandboxURL"? SandboxURL is a const, can't you simply use the const directly?

Reply:
For the same reason, I think we can eliminate Config.IsProduction

Reply:
We could, but the validator is now hardcoded to the const (which is really not an issue). I want to enhance the tests by making them rely less on external sources that we don't have control over. In the tests `validator_test.go`, we want to be able to set the SandboxURL to our own http test server. 

I would like to make this a bit clearer by renaming `URL` to `ProductionURL` so there is less confusion

Reply:
Also it was the only way I could keep these tests at 100% coverage. I'm always open to hearing of other ways :)